pan3d/serve_viewer.py

```diff
@@ -19,8 +19,8 @@ def main():
     builder = DatasetBuilder(dataset_path=args.dataset, pangeo=args.pangeo)
     if args.config_path:
         builder.import_config(args.config_path)
-    builder.viewer.server.start(debug=args.debug)
+    builder.viewer.layout.server.start(debug=args.debug)
```

Comment:
You could remap the `server.start` to `viewer.start` to make it shorter

Reply:
Good idea. Changed in [b87a945](https://github.com/Kitware/pan3d/pull/59/commits/b87a9450a989c0c672d41afe7d852b030323d219)